src/components/alert-notifications/notification-item.vue

```diff
@@ -32,11 +32,10 @@ function onEnter(el: Element, done: () => void) {
   >
     <div
       v-if="show"
+      v-html="props.notification.message"
       class="lux-alert"
       :class="`lux-${props.notification.type}`"
       role="alert"
-    >
-      {{ props.notification.message }}
```

Comment:
Not sure if we should remove simple text completely. Maybe we could make html optional?

Reply:
Why so? Letting `v-dompurify-html` handle the case is simpler I think. But maybe I misunderstood?
I leave it as is for now, feel free to add a new param later.